src/Resources/views/debug_pages/project_page_type_links_and_info.html.twig

```diff
@@ -0,0 +1,29 @@
+
+<table border="1">
```

Comment:
Is hier nog een eenvoudige bootstrap saus overheen te gooien zodat het er nog wat modern uitziet :eye: